doc/advanced.rst

```diff
@@ -64,6 +64,27 @@ its reference
 it generated has the name ``sphx_glr_plot_0_sin_001.png``
 and its thumbnail is ``sphx_glr_plot_0_sin_thumb.png``
 
+You can also include part of a gallery script elsewhere in your documentation
+using the `literalinclude directive <https://www.sphinx-doc.org/en/master/usage/restructuredtext/directives.html#directive-literalinclude>`_, in order to limit code duplication::
+
+   .. literalinclude:: ../examples/plot_0_sin.py
+      :language: python
+      :start-after: # License: BSD 3 clause
+      :end-before: # To avoid matplotlib
+
+The above directive inserts the following block:
+
+.. literalinclude:: ../examples/plot_0_sin.py
+    :language: python
+    :lines: 43-51
```

Comment:
I expected this to be a verbatim copy of the above but it's not, why use line numbers here but before and after above?

Reply:
oops, good catch, I modified one but not the other